app/src/main/java/com/example/mycalculator/symbols/OperatorsSymbols.java

```diff
@@ -0,0 +1,8 @@
+package com.example.mycalculator.symbols;
+
+public class OperatorsSymbols {
```

Comment:
you can change this class name to ConstantsUtils and use this class for the new needed values

Reply:
Done